pages/index.js

```diff
@@ -14,6 +14,29 @@ import includes from 'lodash/includes';
 import Link from 'next/link';
 
 export default class extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      // default to changelog if no forumlink is found
+      releaseNote: {
+        link: '/changelog'
+      }
+    }
+  }
+
+  componentDidMount() {
+    fetch(`https://forums.resin.io/c/etcher.json`)
+    .then((res) => res.json())
+    .then((releaseNotes) => {
+      return releaseNotes.topic_list.topics.find(topic => ( topic.title.includes(locals.version)))
```

Comment:
Good point, we previously built the link assuming a format like `new-release-<v>` but at some point we changed the format which borked the link.

Maybe we should make the check more strict? Like check for `new` `release` and `<version>` ? Still might end up breaking though. 

We could also manually add the link, but then you have post of the forum before publishing the site.